chapter07/red/exercise1.pl

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+
+use File::Find;
+use Date::Calc qw( Day_of_Week Localtime Add_Delta_Days Mktime );
```

Comment:
Date::Calc is a bit old and ugly, why not try one of those fancy new time classes?